### PR TITLE
Fixed bold text in post migration dialog.

### PR DESCRIPTION
**Pull request by tonyanziano**
Fix for #1167

### CODE REVIEW

File: packages/app/client/src/ui/dialogs/postMigrationDialog/postMigrationDialog.tsx [hunk 1]
```diff
@@ -47,8 +48,11 @@ export class PostMigrationDialog extends React.Component<PostMigrationDialogProp
     return (
       <Dialog cancel={ this.onClose } title="Migration complete!">
         <p>
-          { 'We’ve copied your bot endpoints from Emulator v3 and saved them as <strong>.bot files</strong>. ' +
-          'A <strong>.bot file</strong> stores metadata about different services your bot consumes and enables you ' +
+          { 'We’ve copied your bot endpoints from Emulator v3 and saved them as ' }
+          <strong className={ styles.boldText }>.bot files</strong> 
+          { '. A ' }
+          <strong className={ styles.boldText }>.bot file</strong>
+          { ' stores metadata about different services your bot consumes and enables you ' +
```
Comment (cwhitten) [inline packages/app/client/src/ui/dialogs/postMigrationDialog/postMigrationDialog.tsx]: why the need to concatenate two different strings?

File: packages/app/client/src/ui/dialogs/postMigrationDialog/postMigrationDialog.tsx [hunk 2]
```diff
@@ -47,8 +48,11 @@ export class PostMigrationDialog extends React.Component<PostMigrationDialogProp
     return (
       <Dialog cancel={ this.onClose } title="Migration complete!">
         <p>
-          { 'We’ve copied your bot endpoints from Emulator v3 and saved them as <strong>.bot files</strong>. ' +
-          'A <strong>.bot file</strong> stores metadata about different services your bot consumes and enables you ' +
+          { 'We’ve copied your bot endpoints from Emulator v3 and saved them as ' }
```
Comment (cwhitten) [inline packages/app/client/src/ui/dialogs/postMigrationDialog/postMigrationDialog.tsx]: what is the i18n story with the Emulator? English only for now?

File: packages/app/client/src/ui/dialogs/host/host.tsx [hunk 3]
```diff
@@ -34,25 +34,28 @@
...
     super(props);
   }
 
   public componentDidMount() {
     this._hostRef.addEventListener('dialogRendered', this.initFocusTrap);
   }
 
+  public componentWillUnmount() {
+    this._hostRef.removeEventListener('dialogRendered', this.initFocusTrap);
+  }
```
Comment (a-b-r-o-w-n) [inline packages/app/client/src/ui/dialogs/host/host.tsx]: This appears to be unrelated to #1167. Can you provide some context about this change?

Comment (tonyanziano) [reply packages/app/client/src/ui/dialogs/host/host.tsx]: Noticed that the component was never removing the event listener so I added it. I can split it into a different PR if necessary.

Comment (justinwilaby) [reply packages/app/client/src/ui/dialogs/host/host.tsx]: @a-b-r-o-w-n - We often address tech debt in this way. The "see a weed, pull it now" strategy...
